indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -421,6 +421,7 @@ public long push() throws IOException
       case "hdfs":
       case "viewfs":
       case "maprfs":
+      case "s3a":
```

Comment:
Any reason s3a is here instead of in the "s3_zip" section? I think s3a should be treated like s3 deep storage, not hdfs.